src/franchise.js

```diff
@@ -58,7 +59,19 @@ class Voter {
    }
 }
 
+function computeVoteValue(voteBuffer) {
+   const voteValueHash = crypto.createHash("sha256")
+      .update(voteBuffer)
+      .digest("hex");
+   const voteValue = [
+      BigInt("0x" + voteValueHash.slice(0, 32).match(/.{2}/g).reverse().join("")), // little-endian BigInt representation
```

Comment:
:woozy_face:

Reply:
is in this room the meeting of js fan club? ^^